packages/cli/src/commands/generate/index.ts

```diff
@@ -0,0 +1,33 @@
+import { QueryEngine } from '@synthql/backend';
+import { Pool } from 'pg';
+
+interface GenerateSchemaOptions {
+    connectionString: string;
+    out?: string;
+    defaultSchema?: string;
+    schemas?: string[];
+}
+
+export const generate = async ({
+    connectionString,
+    out,
+    defaultSchema,
+    schemas = [],
+}: GenerateSchemaOptions) => {
+    const pool = new Pool({
+        connectionString,
+    });
+
+    const queryEngine = new QueryEngine({
+        pool,
+        schema: defaultSchema,
+    });
+
+    const result = await queryEngine.generateSchema({
+        out,
```

Comment:
Given that `out` is a folder, you should append a file name here, such as `schema.ts`